mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_backup_policy.go

```diff
@@ -279,7 +280,7 @@ func expandPolicies(d *schema.ResourceData) []matlas.Policy {
 		policy := v.(map[string]interface{})
 		policies[k] = matlas.Policy{
 			ID:          policy["id"].(string),
-			PolicyItems: expandPolicyItems(policy["policy_item"].([]interface{})),
+			PolicyItems: expandPolicyItems(policy["policy_item"].(*schema.Set).List()),
```

Comment:
Was surprised to see a change in the deprecated resource other than the message, is this related to the SDK upgrade?

Reply:
Good eye, I didn't realized that, I will roll back since I created another function for this resource cloud backup schedule